### DISCUSSION TITLE
Figma-to-XAML Plugin Won't Recognize Page With Swipable Content

### DISCUSSION THREAD

**Discussion by elibroftw** [unoplatform/uno#12156] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/21298211/236059107-ae65216e-ae91-4cdd-8c48-1b7a6ae5eca8.png)

**Comment by elibroftw** [comment]
Had to detach instance on the swipe control list. the plugin should tell me that though.

**Reply by elibroftw** [reply]
Doesn't work anymore again.

**Comment by elibroftw** [comment]
![image](https://user-images.githubusercontent.com/21298211/236063473-fdabbd69-dde1-4084-b9f2-4ddfdd0bd25b.png)
Doesn't work for template in the getting started either.

**Reply by elibroftw** [reply]
Works in the community file but not the one I copied originally.

**Comment by elibroftw** [comment, marked as answer]
To fix, need to migrate to the new community material toolkit

1. Rename old project to "projectOld"
4. On left side, select page with your design
5. Ctrl + A so that all your designs are selected
![image](https://user-images.githubusercontent.com/21298211/236069481-8ffdc286-8ccf-4ee1-8952-886bb7ca5f37.png)
6. Ctrl + C to copy
7. Open [Uno Platform Material Toolkit](https://www.figma.com/community/file/1110792522046146058) in Figma
8. Rename to project
9. Create new page "AppName app"
10. Ctrl + V
11. Replace any broken components used in old design with the new ones under assets. I suggest pasting the asset into an open area, editing it so that it looks the same as the old broken component, and then lastly pasting next to the broken component before deleting the broken component.

**Comment by carldebilly** [comment]
The Figma plugin API were crashing, preventing the plugin from generation a XAML for the selection. An updated version (v2.5.2) of the plugin able to handle this situation will be deployed in the next few hours.
